plotly-plot-behavior.js

```diff
@@ -189,6 +189,17 @@
         return self.fire('plotly-unhover', {data: data});
       };
 
+      /**
+       * Custom plotly-specific event for binding to zoom events.
+       * Fires after zoom.
```

Comment:
Does any other operation cause a relayout? Please be more exact here, since this event will be used for more than just your use cases. That is, why doesn't Plotly call it `plotly_zoom`? Whatever they use it for is what we should describe it as being used for.